core/src/test/resources/flows/valids/webhook-with-condition.yaml

```diff
@@ -0,0 +1,15 @@
+id: webhook-with-condition
+namespace: io.kestra.tests
```

Comment:
```suggestion
namespace: dev
```

Reply:
We always use this namesapce in our tests, this is not something visible to users

Reply:
We always use this namesapce in our tests, this is not something visible to users